tornado/options.py

```diff
@@ -41,6 +41,13 @@ def connect():
     # or
     tornado.options.parse_config_file("/etc/server.conf")
 
+.. note:
+
+   When using tornado.options.parse_config_file, the only options that are set
+   are ones that were previously defined with either tornado.options.parse_command_line
+   or tornado.options.define. tornado.options.parse_config_file is used mainly as a 
+   means to offer overriding settings for different environments.
```

Comment:
Is it for overriding? I think that if both command line and config file are used the command line should override what's in the file instead of the other way around. But then I've always thought it was weird to use both and apps would choose one or the other (or perhaps use both but only pass the location of a config file on the command line. Doing it in this order would make the file override the command line, but it would be strange to pass the same option in both places if you use this style)